### PR TITLE
fix(wrapper): Fix issue with specifying versions

### PR DESCRIPTION
**Pull request by brianseitel**
### What

Fixes a bug with specifying versions. Previous wrapper versions were allowing you to customize the `v1` part, but that's not what we wanted to do. 

### Why

Fixin' bugs is all in a day's work.

### CODE REVIEW

**Comment by mgartner** [comment]
Talked IRL - we should not set the `Lob-Version` header if a version is not speicified.

**Comment by brianseitel** [comment]
Updated.

**Comment by mgartner** [comment]
LGTM

File: tests/Lob/Tests/LobTest.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ protected function setUp()
 
     public function testVersionDefaultValueIsV1()
```
Comment (robinjoseph08) [inline tests/Lob/Tests/LobTest.php]: Can you rename this to `testVersionDefaultValueIsNull`?

**Comment by robinjoseph08** [comment]
Small comment but LGTM

**Comment by brianseitel** [comment]
@robinjoseph08 changed

**Comment by robinjoseph08** [comment]
LGTM :+1:

**Pull request merged**